scripts/plugins/extauth-hook-AD.py

```diff
@@ -37,23 +37,27 @@
 
 def setup_logger():
     """Helper function setup logger"""
-    log = logging.getLogger()
+    addr = "/dev/log"
+
     logging.basicConfig(
         format='%(asctime)s %(levelname)s %(name)s %(funcName)s %(message)s', level=logging.DEBUG)
+    log = logging.getLogger()
+
+    if not os.path.exists(addr):
+        log.warning("{} not available, logs are not redirected".format(addr))
```

Comment:
How will this get logged if the location is not available?

Reply:
yes, it is never happens in xenserver.  it just print to stdout during CI. (container)